source/layouts/layout.erb

```diff
@@ -7,11 +7,17 @@
 	<%= partial "/layouts/partials/footer" %>
 
 	<!--   Core JS Files   -->
-	<script src="/assets/js/core/jquery.3.2.1.min.js" type="text/javascript"></script>
-	<script src="/assets/js/core/tether.min.js" type="text/javascript"></script>
-	<script src="/assets/js/core/bootstrap.min.js" type="text/javascript"></script>
+	<script src="https://code.jquery.com/jquery-3.3.1.min.js" integrity="sha256-FgpCb/KJQlLNfOu91ta32o/NMZxltwRo8QtmkMRdAu8=" crossorigin="anonymous"></script>
+	<script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.7/umd/popper.min.js" integrity="sha384-UO2eT0CpHqdSJQ6hJty5KVphtPhzWj9WO1clHTMGa3JDZwrnQq4sF86dIHNDz0W1" crossorigin="anonymous"></script>
+	<script src="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/js/bootstrap.min.js" integrity="sha384-JjSmVgyd0p3pXB1rRibZUAYoIIy6OrQ6VrjIEaFf/nJGzIxFDsf4x0xIM+B07jRM" crossorigin="anonymous"></script>
```

Comment:
why are we mixing sha256 and sha384?

Reply:
I don't know, They are copy pasted from the bootstrap website, I'm guessing jquery only use sha256 where cloudflare and stackpath in force sha384